static/docs/commands-reference/status.md

```diff
@@ -91,15 +91,14 @@ cache. For the typical process to update workspaces, see
   name defined using the `dvc remote` command. Implies `--cloud`.
 
 - `-a`, `--all-branches` - compares cache content against all Git branches.
-  Instead of checking just the currently checked out workspace, it checks
-  against all other branches of this workspace. The corresponding branches are
-  shown in the status output. Applies only if `--cloud` or a remote is
-  specified.
+  Instead of checking just the workspace, it checks against all other branches
+  of this workspace. The corresponding branches are shown in the status output.
```

Comment:
of his workspace -> of this repo

Reply:
i'm not sure if we need use "against" as well. It just runs the same status command for all branches.

Reply:
Updated, please check. I removed one of the `against` words but left the first one which I think make sense.